src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -11,16 +11,21 @@
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(UseContractReferences)' == 'true'" >
```

Comment:
I see. Thanks for the explanation. I thought it was the old variable which we want it to be true in default case.

So by default UseContractReferences would be empty (''). If someone set UseContractReferences=false, the project wouldn't build. Just wanted to confirm this is the expected behavior.